src/openseadragon.js

```diff
@@ -249,6 +249,11 @@
   *     image though it is less effective visually if the HTML5 Canvas is not
   *     availble on the viewing device.
   *
+  * @property {Number} [smoothTileEdgesMinZoom=1.1]
+  *     A zoom percentage ( expressed as a number between 0 and 1 ) of the highest
```

Comment:
I think saying "between 0 and 1" is confusing because even the default is out of that range. How about "where 1 is 100%"?